### ISSUE TITLE
Unit tests not run during build

### ISSUE THREAD

**Issue by jjhelmus** [conda-forge/scikit-image-feedstock#23]
Although the recipe has the logic to run the unit tests via `sys.exit(skimage.test(verbose=True))`, the unit tests are not run during the build.  

[For example](https://travis-ci.org/conda-forge/scikit-image-feedstock/jobs/385904093#L868-L877):

```
===== testing package: scikit-image-0.14.0-py36hfc679d8_1 =====
running run_test.py
============================= test session starts ==============================
platform darwin -- Python 3.6.5, pytest-3.6.0, py-1.5.3, pluggy-0.6.0 -- /Users/travis/miniconda3/conda-bld/scikit-image_1527717783343/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/python
cachedir: .pytest_cache
rootdir: /Users/travis/miniconda3/conda-bld/scikit-image_1527717783343/test_tmp, inifile:
plugins: cov-2.5.1
collecting ... 
========================= no tests ran in 0.00 seconds =========================
ERROR: file not found: skimage
```

I believe the upstream source needed to be patched to have the first argument passed to pytest.main be the package directory rather than 'skimage':

``` diff
diff --git a/skimage/__init__.py b/skimage/__init__.py
index d29ae82..aa9ddf0 100644
--- a/skimage/__init__.py
+++ b/skimage/__init__.py
@@ -81,7 +81,7 @@ else:
         """Run all unit tests."""
         import pytest
         import warnings
-        args = ['skimage']
+        args = [pkg_dir]
         if verbose:
             args.extend(['-v', '-s'])
         if doctest:
```

I'll raise an issue upstream.

**Comment by hmaarrfk** [comment]
I think the line 
```
  commands:
    - pytest --pyargs skimage
```
should just be added at the bottom getting rid of an other file.

**Comment by ocefpaf** [comment]
I guess that this is fixed in #28 

Please re-open if not.
